tiptapy/__init__.py

```diff
@@ -63,9 +63,9 @@ def inner_render(self, node) -> str:
                            if k not in special_attrs_map and v.strip()
                            )
         html = f"<img {attrs_s}>"
-        if attrs.get('caption').strip():
+        if 'caption' in attrs and attrs.get('caption').strip():
```

Comment:
@Pradhvan `if 'caption' in attrs and attrs['caption'].strip():` or 'if attrs.get('caption', '').strip():'

Reply:
@gauravr `if attrs.get('caption', '').strip():` this is more Pythonic. Updating the PR. 😄